src/icons/crypto/AfricaRare.tsx

```diff
@@ -0,0 +1,884 @@
+      <path fill="#E46D21" d="m1703 788 1 4 2 1-3 4v-2l-4 2-3-2 2-1v-2l-2-1Z" />
+      <path fill="#EE8231" d="m529 894 4 1 3 5v2h-5l-2-3Z" />
+      <path fill="#DF6218" d="m1213 503 1 4-2 1-1 3-5-1-2-2 3-1-3-3 8 1Z" />
+      <path fill="#DE6117" d="m440 704 4 4 1 3h-3l-2 5-1-2 1-4h-2l-2-5 3 1Z" />
+      <path fill="#DF6218" d="M1676 1077h6l-3 5h-3v6h-1v-7l-3-1 3-1Z" />
+      <path
+        fill="#DB5B13"
+        d="m400 809 1 3 2 1h-3v3l9 2-4 1-7-1v-2l-2-1 2-4h2Z"
+      />
+      <path
+        fill="#DA5810"
+        d="M1195 548h2l1 6h2l2-4v6l2 1-2 2-3 1-2-2h3l-2-2h-1Z"
+      />
+      <path fill="#E1671C" d="m1413 1339 5 2 1 2-5-1-9 9v-3Z" />
+      <path fill="#E2681D" d="m1578 947 7 1v3l4-1-1 2h-2l-1 2-5-2Z" />
+      <path
+        fill="#D8530D"
+        d="m340 1146 2 1-3 1-1 3v-2l-2 2v5l-2-4v-2l-6-2 2-1h4Z"
+      />
+      <path fill="#E2681D" d="m1248 525 3 1-1 7-1-4h-3l1 7-2-2v-7Z" />
+      <path
+        fill="#E2691D"
+        d="M1365 473h1l1 10h-3l2 4-5-1v-5h4ZM839 1759l1 2v6l2 3-4-2v-3h-2l-1-5Z"
+      />
+      <path
+        fill="#D9550E"
+        d="m1717 700 2 2 2 5 2 2-2 3v-2l-7-1v-1h5v-2l-2-1Z"
+      />
+      <path
+        fill="#D8540E"
+        d="m1140 532 1 2-1 6-1-3h-3l2 7-3-3v-5l-2-1h2l1-2 3 3Z"
+      />
+      <path fill="#E46D20" d="M240 826h1l-1 8h2v4h-3l-1-2-3-2h4Z" />
+      <path fill="#E46C20" d="m1489 776 3 4v2h6v1h-5v3l-3 1-1-3 1-3Z" />
+      <path fill="#E67123" d="m1148 1764 4 1 3 5-5-1-1 2-2-4Z" />
+      <path fill="#EF8533" d="M634 686h2l1 6h-5l1-5Z" />
+      <path fill="#E36A1E" d="m396 678 5 4 4 2h-4v2l-4-1Z" />
+      <path
+        fill="#D8540D"
+        d="m1223 252 3 2h-3v3h3l-1 2-2-1 1 6-2-1-1-6-2-1h3Z"
+      />
+      <path
+        fill="#D7520C"
+        d="m940 1524 2 1-4 5h3l-1 3v-2h-3l-1 2v-2l-4-1 4-1Z"
+      />
+      <path fill="#D8550E" d="m328 867 5 4h-2v3l2 1-2 1-1 5h-1l-1-8h2Z" />
+      <path fill="#D8530D" d="m1128 1671 3 2-2 2-3-1-11 1v-1Z" />
+      <path fill="#E77426" d="M277 1004v3l-3 1-1 4h-3l1-4Z" />
+      <path fill="#E97829" d="m1603 803 1 3-4 5-1 5h-1v-6l2-2 1-4Z" />
+      <path fill="#E77325" d="M1130 530h8v2l2 1-1 3-3-1v-3l-6-1Z" />
+      <path fill="#D7520C" d="m1119 473 1 4 2 1h-2v3l2 1-4 1-3-2 1-3v2h2Z" />
+      <path fill="#E36C20" d="M937 1531h3l3 7 1 2-4-2v-3h-3Z" />
+      <path fill="#E57023" d="m1112 471 4 2v3h2l1 5-3-1-3-7Z" />
+      <path fill="#E77325" d="m838 238 2 4 2 3-6-2-1-4Z" />
+      <path fill="#EA7829" d="m460 1004 8 5v3h-3l-1-3-4-3Z" />
+      <path fill="#D8540D" d="m1702 792 4 1-3 4v-2l-4 2-3-2 6-1Z" />
+      <path fill="#EB7B2B" d="M1407 581h9l-3 2-6 1Z" />
+      <path fill="#E77426" d="m694 1547 3 1-7 6 1-4Z" />
+      <path fill="#E57023" d="m1424 1223 2 1-7 6 1-4Z" />
+      <path fill="#FDA34A" d="m1719 698 3 4v2l-3 1-2-6Z" />
+      <path fill="#E57023" d="M439 690h2l-2 4-4 4 1-5Z" />
+      <path
+        fill="#E97728"
+        d="m1489 524 2 4-1 2-4-1ZM1397 386l6 2 1 4-3 1-1-4Z"
+      />
+    </SvgIcon>
```

Comment:
Can you merge all these layers into a single shape in Figma, and resize the whole icon to a 24x24 size? This one is way too heavy. I’m out today but maybe Daniil can help